recipes/moorpy/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "MoorPy" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nrel/MoorPy/archive/v{{ version }}.tar.gz
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pCrunch-{{ version }}.tar.gz
+  sha256: d6a408844357f5e8a318875db110047b86a4a3956ea1659f8fbdeb1ec685cfb2
```

We prefer PyPI URLs.

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/MoorPy-{{ version }}.tar.gz
  sha256: 959c4cbcf0c7cf1d0ec9acc44efecc888c221fd6e40c0a589aa7292847ad4743
```